docs/README.md

```diff
@@ -75,214 +75,97 @@ shown earlier.  Consult the [troubleshooting
 guide](https://github.com/ibm-functions/shell/blob/master/npm.md) if
 your installation fails.
 
-### Tour of the programming shell
-
-At the end of the installation, you can run the programming shell form your terminal. It is
-typically installed in `/usr/local/bin/fsh`.
-
-```
-$ fsh
-Welcome to the IBM Cloud Functions Shell
-
-Usage information:
-fsh about                                    [ Display version information ]
```

Comment:
It might be a nice feature of the shell to generate a help doc and post it online @starpit.